cluster-scope/base/operators.coreos.com/subscriptions/mongodb-operator/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: mongodb-operator
+  namespace: openshift-operators
+spec:
+  Channel: alpha
+  installPlanApproval: Automatic
+  name: mongodb-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: mongodb-operator.v0.3.0
```

Comment:
I don't think you want this line.

Reply:
Good call @computate, removed startingCSV